<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_FirstPage</class>
 <widget class="QWizardPage" name="UI_FirstPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <widget class="QRadioButton" name="btn_create">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Create a new applcation tamplate</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="btn_modify">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Create from an existing application template</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editFolder">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
